bok_choy/page_object.py

```diff
@@ -2,21 +2,28 @@
 Base implementation of the Page Object pattern.
 See https://code.google.com/p/selenium/wiki/PageObjects
 """
-
-import logging
-import socket
-import urlparse
 from abc import ABCMeta, abstractmethod, abstractproperty
-from collections import defaultdict
+from collections import defaultdict, namedtuple
 from functools import wraps
 from contextlib import contextmanager
+import json
+import logging
+import os
+import socket
 from textwrap import dedent
+import urlparse
+
+import requests
 from selenium.common.exceptions import WebDriverException
 
 from .query import BrowserQuery
 from .promise import Promise, EmptyPromise, BrokenPromise
 
 
+AXS_FILE = 'vendor/axs_testing.js'
+AuditResults = namedtuple('AuditResults', 'errors, warnings')
```

Comment:
Oh, both formats mean the same time. I just picked one. Unless that's not what you meant?
https://docs.python.org/2/library/collections.html#collections.namedtuple
> The field_names are a sequence of strings such as ['x', 'y']. Alternatively, field_names can be a single string with each fieldname separated by whitespace and/or commas, for example 'x y' or 'x, y'.